### PR TITLE
Fixes Issue #5 

### PR DESCRIPTION
**Pull request by YBirhane**

### CODE REVIEW

File: ac1/_sources/introduction_to_visualizations/summary.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.. Copyright (C)  Google, Runestone Interactive LLC
+   This work is licensed under the Creative Commons Attribution-ShareAlike 4.0
+   International License. To view a copy of this license, visit
+   http://creativecommons.org/licenses/by-sa/4.0/.
+
+.. image:: figures/summary.png
+   :align: center
+   :alt: A line graph comparing number of likes over time of two posts.
+   
+.. _ Location of summary document: shorturl.at/nswJ5
```
Comment (raskutti) [inline ac1/_sources/introduction_to_visualizations/summary.rst]: Quick question: What does the _ before "Location of summary document" do?

Comment (raskutti) [reply ac1/_sources/introduction_to_visualizations/summary.rst]: @YBirhane

Comment (YBirhane) [reply ac1/_sources/introduction_to_visualizations/summary.rst]: It just has the google docs location of all the summaries we made (and we made the link publicly accessible).

Comment (YBirhane) [reply ac1/_sources/introduction_to_visualizations/summary.rst]: I removed the underscore!

**Pull request merged**